### ISSUE TITLE
MongoDB disk usage?

### ISSUE THREAD

**Issue by upklio** [bitpay/bitcore#2835]
I see MongoDB is required. How is it used? I'm plan to configure my btc / bch nodes to store their data in a mounted volume (600gb) so there's enough space.

But what about MongoDB, that only has 80gb disk-space, what is it used for and what's stored within? How much space does it need?

**Comment by micahriggan** [comment]
Mongodb is used to index the data from the bitcoin nodes. The amount of space used will be greater than the size of a fully synced bitcoin data directory.

**Comment by upklio** [comment]
So let's say bch + btc data amount to 600gb. Mongodb will need another 600gb+!?

Not sure I fully understand based on what I'm seeing in my sync process.

I'm currently syncing btc + bch and here's the stats so far:
```
173G	-datadir=/mnt/bitcoin_cash
182G	-datadir=/mnt/bitcoin
```

mongodb:
```
368M	/var/lib/mongodb
```

Doesn't look like mongo's going to get much bigger?

**Comment by makabress** [comment]
MongoDB database size for btc only is 610Go ,Btc data folder 330Go so you need at last 1To

**Comment by JoePotentier** [comment]
I used a 1TB drive and it filled up in less than a day while. I also only synced BTC and didn't ever configure for BCH. Am I doing something wrong?

**Comment by micahriggan** [comment]
@JoePotentier Did you sync your bitcoin node to that same drive? If so, do you have TxIndex on? That would take up a large amount of additional space.

**Comment by JoePotentier** [comment]
I did, and yes. Set this up to sync before bed and woke up to it being full. What specs are recommended for full sync? Does a pruned node work?

**Comment by micahriggan** [comment]
Yeah pruned works. Bitcore has it's own tx index. That should give you back some space.
